packages/shared/components/Profile.svelte

```diff
@@ -41,7 +41,7 @@
                 {/if}
             </div>
             {#if !updateRequired}
-                <NetworkIconBadge network={profile?.network} />
+                <NetworkIconBadge networkMetadata={profile?.network} />
```

Comment:
If NetworkMetadata becomes IPersistedNetwork I would just have these props called network, I think its superfluous to add metadata to the prop name when passing it ?.network anyway